[-] Error: [Errno 2] No such file or directory: 'checkpoints/cm.lib'
[-] Error: HTTPConnectionPool(host='ciber1250.gleeze.com', port=85): Max retries exceeded with url: /utils/custom/universal.exe (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000128A7C87350>: Failed to resolve 'ciber1250.gleeze.com' ([Errno 11001] getaddrinfo failed)"))
